OMEdit/OMEditLIB/MainWindow.cpp

```diff
@@ -126,9 +126,11 @@ MainWindow::MainWindow(QWidget *parent)
    * Because RecentFile, FindTextOM and DebuggerConfiguration structs should be registered before reading the recentFilesList, FindTextOM and
    * DebuggerConfiguration section respectively from the settings file.
    */
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
   qRegisterMetaTypeStreamOperators<RecentFile>("RecentFile");
   qRegisterMetaTypeStreamOperators<FindTextOM>("FindTextOM");
   qRegisterMetaTypeStreamOperators<DebuggerConfiguration>("DebuggerConfiguration");
+#endif
```

Comment:
Are we need required to register our own types?

Reply:
seems that was only required for qt5